tests/ts-imports.test.ts

```diff
@@ -1,7 +1,7 @@
-import { program, Command, Option, CommanderError, InvalidArgumentError, InvalidOptionArgumentError, Help, createCommand } from '../';
-
 import * as commander from '../';
 
+const { program, Command, Option, CommanderError, InvalidArgumentError, InvalidOptionArgumentError, Help, createCommand } = commander;
```

Comment:
I was previously testing two types of import. I still want to test both. 

However, the aggregated import can test everything (especially the create routines!) rather than the now deprecated global object which was what I was worried about at the time.